CMakeLists.txt

```diff
@@ -237,24 +238,25 @@ endif()
 
 ########################################
 # Install
-
-install(TARGETS nix nix-tool
-        LIBRARY DESTINATION ${LIB_INSTALL_DIR}
-        ARCHIVE DESTINATION ${LIB_INSTALL_DIR}
-        FRAMEWORK DESTINATION "/Library/Frameworks"
-        RUNTIME DESTINATION bin)
-install(DIRECTORY include/ DESTINATION ${INCLUDE_INSTALL_DIR})
-install(DIRECTORY ${CMAKE_BINARY_DIR}/include/ DESTINATION ${INCLUDE_INSTALL_DIR})
+if(NOT ${BUILD_STATIC})
+  install(TARGETS nix nix-tool
+    LIBRARY DESTINATION ${LIB_INSTALL_DIR}
+    ARCHIVE DESTINATION ${LIB_INSTALL_DIR}
+    FRAMEWORK DESTINATION "/Library/Frameworks"
+    RUNTIME DESTINATION bin)
+  install(DIRECTORY include/ DESTINATION ${INCLUDE_INSTALL_DIR})
+  install(DIRECTORY ${CMAKE_BINARY_DIR}/include/ DESTINATION ${INCLUDE_INSTALL_DIR})
+endif()
 
 #pkg-config support
 configure_file(${CMAKE_SOURCE_DIR}/nix.pc.in ${CMAKE_BINARY_DIR}/nix.pc)
-install(FILES ${CMAKE_BINARY_DIR}/nix.pc DESTINATION ${LIB_INSTALL_DIR}/pkgconfig)
-
+if(NOT EXISTS "${LIB_INSTALL_DIR}/pkgconfig/nix.pc")
```

Comment:
hmm, if we wanna package libnixio (debian, fedora) and ship the static and non-static version, then this might now work, because you build in a build-root that is separate from there rest of the system.
Maybe better would be a config option?